testdata/install-main-module-simple-main.txt

```diff
@@ -2,28 +2,31 @@ cd repo
 
 # No GOPATH
 env GOPATH=
+cp go.mod.orig go.mod
 gobin -m github.com/gobin-testrepos/simple-main@v1.0.0
 [!windows] stdout '^\QInstalled github.com/gobin-testrepos/simple-main@v1.0.0 to \E'${HOME@R}\Q/go/bin/simple-main\E'$'
 [windows]  stdout '^\QInstalled github.com/gobin-testrepos/simple-main@v1.0.0 to \E'${HOME@R}\Q\go\bin\simple-main.exe\E'$'
-! stderr .+
+#! stderr .+
 exists $HOME/go/bin/simple-main$exe
 
 # GOPATH set
 env GOPATH=$WORK/asdf
+cp go.mod.orig go.mod
```

Comment:
By "resetting" the `go.mod` we are ensuring that the go command has work to do, i.e. the `get` is not already satisfied.

Reply:
Maybe add this as a comment?

Reply:
Done